.dockerignore

```diff
@@ -1,2 +1,7 @@
 target
```

Comment:
Do we need this file? Does it do anything?

Reply:
Hmm, I cannot see where it is used. Probably, we can just remove it away